lib/undercover/gem_version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Undercover
-  VERSION = '1.1.0'
+  VERSION = '1.1.1'
```

Comment:
If we change the supported Ruby versions as recommended, it should be a major version change as it is a breaking change:

```suggestion
  VERSION = '2.0.0'
```